docs/python/testing.md

```diff
@@ -153,41 +158,32 @@ Test discovery applies the discovery patterns for the current framework (which c
 
 > **Tip**: Sometimes tests placed in subfolders aren't discovered because such test files cannot be imported. To make them importable, create an empty file named `__init__.py` in that folder.
 
-If discovery succeeds, the status bar shows **Run Tests** instead:
-
-![Status bar showing successful test discovery failed](images/testing/discovery-succeeded-status-bar.png)
-
-If discovery fails (for example, the test framework isn't installed), you see a notification on the status bar. Selecting the notification provides more information:
+[comment]: <TODO: update text below>
 
-![Status bar showing that test discovery failed](images/testing/discovery-failed-status-bar.png)
+If discovery succeeds, you will see tests displayed in the test explorer:
 
-Once VS Code recognizes tests, it provides several ways to run those tests as described in [Run tests](#run-tests). The most obvious means are CodeLens adornments that appear directly in the editor and allow you to easily run a single test method or, with unittest, a test class:
-
-![Test adornments that appear in the VS Code editor for unittest code](images/testing/editor-adornments-unittest.png)
+![The VS Code Test Explorer for Python tests](images/testing/test-explorer.png)
 
-![Test adornments that appear in the VS Code editor for pytest code](images/testing/editor-adornments-pytest.png)
+If discovery fails (for example, the test framework isn't installed), you will see an error message displayed on the test explorer. You can click on the console icon at the top-right of the test explorer to see what the entire error message in the output.
```

Comment:
Discovery errors are in the python logs. Run errors are in the Run console logs.